coops-ui/src/mixins/resource-lookup-mixin.ts

```diff
@@ -40,4 +40,40 @@ export default class ResourceLookupMixin extends Vue {
       }
       return ''
     }
+
+    /**
+     * Method to validate directors on edit/cease and return any warning messages.
+     *
+     * @return the compliance message or null (if the configuration has been loaded).
+     */
+    directorWarning (directors: Array<any>): Object {
+      const configSection = this.configObject.flows.find(x => x.feeCode === 'OTCDR').warnings
+      let errors = []
+      // FUTURE: Too much code for this. Can be condensed and made more reusable.
+      if (configSection.bcResident != null) {
```

Comment:
Would it be better to simply check is bcResident exists? Then you wouldn't have to declare a null property in the config file where it isn't needed (ie, it would be optional).